src/vm/script.rs

```diff
@@ -2460,6 +2460,114 @@ impl<'a> ScriptExecutor<'a> {
                     }
                 }
 
+                ScriptEntry::Opcode(OP::Add) => {
+                    if exec_stack.len() < 2 {
+                        self.state = ScriptExecutorState::Error(
+                            ExecutionResult::InvalidArgs,
+                            (i_ptr, func_idx, op.clone(), exec_stack.as_slice()).into(),
+                        );
+                    }
+
+                    let mut last = exec_stack.pop().unwrap();
+                    *memory_size -= last.size();
+                    let mut second = exec_stack.pop().unwrap();
+                    *memory_size -= second.size();
+
+                    match second.add(&mut last) {
+                        Some(_) => {
+                            *memory_size += second.size();
+                            exec_stack.push(second);
+                        }
+                        None => {
+                            self.state = ScriptExecutorState::Error(
+                                ExecutionResult::InvalidArgs,
+                                (i_ptr, func_idx, op.clone(), exec_stack.as_slice()).into(),
+                            );
+                        }
+                    }
+                }
+
+                ScriptEntry::Opcode(OP::Sub) => {
+                    if exec_stack.len() < 2 {
+                        self.state = ScriptExecutorState::Error(
+                            ExecutionResult::InvalidArgs,
+                            (i_ptr, func_idx, op.clone(), exec_stack.as_slice()).into(),
+                        );
+                    }
+
+                    let mut last = exec_stack.pop().unwrap();
+                    *memory_size -= last.size();
+                    let mut second = exec_stack.pop().unwrap();
+                    *memory_size -= second.size();
+
+                    match second.sub(&mut last) {
```

Comment:
This should be reversed i.e. the second item popped should be subtracted from the first.

Reply:
just to clarify, the exec stack would be the inverse of the input stack. I guess if in the script we have `[4, 2, div]` we'll have in the exec stack as `[div, 2, 4]` so the second item pooped in the op code would be 4. 

Having it in reverse here would mean to divide 2 / 4. Am I correct ?

Reply:
The input stack is spilled into the execution stack of the first frame in a `FIFO` manner. Assuming the first item in the following stack is the leftmost: `[10, 2]` the add instruction would pop both and divide `10` by `2` then push `5` to the execution stack: `[5]`.